.travis.yml

```diff
@@ -27,8 +27,9 @@ matrix:
         apt:
           sources:
             - ubuntu-toolchain-r-test
+            - llvm-toolchain-trusty-5.0
           packages:
-            - clang-format-3.9
+            - clang-format-5.0
```

Comment:
Out of curiosity why did you chose version 5.0 specifically? When .travis and CLANG_FORMAT got out of sync before I had issues with the newer version running on travis, which is why I rolled back to 3.9. But it seems like it accepted this, so guess its fine

Reply:
clang-format-5.0 seems the latest available on both our local machines and on travis.